File: abc161/c/c.go
package main

import "fmt"

func repi(n int64, k int64) int64 {
	mod := n % k
	ans := min(mod, k-mod)
	return ans
}

func main() {
	var n, k int64
	fmt.Scan(&n, &k)
	fmt.Println(repi(n, k))
}

File: abc161/c/c.cpp
#include <iostream>

using namespace std;

long long repi(long long n, long long k) {
  long long mod = n % k;
  long long ans = min(mod, k - mod);

  return ans;
}

int main(void) {
  long long n, k;
  cin >> n >> k;
  cout << repi(n, k) << "\n";

  return 0;
}
